revolv/static/js/base/dashboard.js

```diff
@@ -0,0 +1,38 @@
+$(document).ready(function(){
+	$('.accordion-navigation').click(function(){
+		var active = $(this).find("div").hasClass("active");
+		$('.accordion li div').removeClass("active")
+		if (!active){
+			var liItem = $(this);
+			$('.accordion li div').removeClass("active")
+			var href = $(this).find("div").addClass("active");
+		}
+	});
+
+	$('.status-bar').hover(function(){
+		$(".status-bar").removeClass('active');
+		$(this).addClass('active');
+		$(this).css("background-color", "white");
+		$(this).find(".status-bar-menu-text").css("color", "$revolv-header-color");
+	}, function() {
+		$(".status-bar").removeClass('active');
+		$(this).css("background-color", "");
+		$(this).find(".status-bar-menu-text").css("color", "white");
+	});
+
+	doOverlay();
+
+});
+
+var doOverlay = function(){
+	$(".overlay").each(function(){
+		console.log("overlay");
+		var item = $(this);
+		item.css("width", "100%");
```

Comment:
You could probably just put all these styles in a class in the scss, then add a class here. This would also allow you to use a mixin for linear-gradient, -webkit-linear-gradient, -moz-linear-gradient etc